SSA/Projects/InstCombine/HackersDelight.lean

```diff
@@ -302,7 +302,113 @@ theorem slt_iff_adc :
 
 theorem sle_iff_adc_two_pow_sub_neg_add_two_pow_sub :
     (x ≤ₛ 0) ↔ (BitVec.carry w (2 ^ (w - 1)) (-(x + 2 ^ (w - 1)))) false := by
-  sorry
+<<<<<<< Updated upstream
+  sorry
+=======
```

Comment:
This is called a conflict marker. Git uses this to show when you've had a merge conflict. Just delete this.